File: math/maximum-score-from-removing-stones/maximum-score-from-removing-stones.java
class Solution {
  public int maximumScore(int a, int b, int c) {
    int max = Math.max(Math.max(a, b), c);
    int sum = a + b + c;
    if (sum - max < max) {
      return sum - max;
    }
    return sum / 2;
  }
}

File: math/maximum-score-from-removing-stones/maximum-score-from-removing-stones.ts
function maximumScore(a: number, b: number, c: number): number {
  const max: number = Math.max(a, Math.max(b, c));
  const sum: number = a + b + c;

  if (sum - max < max) {
    return sum - max;
  }
  return Math.floor(sum / 2);
}
